File: tareajava/ejercicio32.java
import java.util.Scanner;
public class ejercicio{
  
  public static void main(string args[]){
    Scaneer entrada=new scanner(system.in);
    float pagahora=0;
    float sueldomensual=0;
    int cantidaddehora=0;

  system.out.println("ingrese el costo por hora: ");
  pagohora=entrada.nextFloat();
  system.out.println("ingrese la cantidad de horas trabajadas en la semana");
  cantidaddehora=entrada.nextInt();

    if (cantidaddehora>40){
      sueldomensual=40*pagohora+2*((cantidadadehora-40)*pagohora);
      system.out.println("el sueldo del trabajador es " + sueldomensual),}else if (cantidaddehora<=40){
      sueldomensual=cantidaddehora*pagohora;
      system.out.println("el sueldo del trabador es "+ sueldomensuel);}else{
      
      }
  }
}

File: EstCondicional/ejercicio32.py
#Declarar Variables
pagohora:float
cantidadhora:int
ingresosemanal:float
#Datos de Entrada
pagohora=float(input("Ingrese el costo por hora:"))
cantidadhora=int(input("Ingrese cantidad horas trabajadas en la semana:"))
#Proceso y Datos de Salida
if cantidadhora>40:
  ingresosemanal=40*pagohora+2*((cantidadhora-40)*pagohora)
else:
  ingresosemanal=cantidadhora*pagohora
#Datos de Salida
print(f"El sueldo del trabajador es {ingresosemanal}")